md/connector-archetype.md

```diff
@@ -0,0 +1,82 @@
+## How to create a custom connector apart from the Bonita Studio
+
+Bonita offers the possibility to create au plug custom connectors, to allow you to connect with any external system you may need.  
+A maven archetype is available on maven central to help you to bootstrap a connector project apart from the Bonita Studio. The source code of the archetype is available [here](https://github.com/bonitasoft/bonita-connector-archetype).
+
+### Prerequisite
+
+ 1. Java must be installed: [https://adoptopenjdk.net/index.html](https://adoptopenjdk.net/index.html)
+ 2.  Maven must be installed: [https://maven.apache.org/install.html](https://maven.apache.org/install.html)
+ 3. Developing a connector apart from the Bonita Studio requires some software development skills. The  archetype offers the possibility to develop the connector in Java, Groovy or Kotlin. Make sure that you are comfortable with at least one of those three languages. 
```

Comment:
```suggestion
 3. Developing a connector outside Bonita Studio requires some software development skills. The  archetype offers the possibility to develop the connector in Java, Groovy or Kotlin. Make sure that you are comfortable with at least one of those three languages. 
```

Reply:
I also used this sentence in the title, it's ok to keep it there?